components/style/properties/longhand/box.mako.rs

```diff
@@ -1083,22 +1084,24 @@ ${helpers.predefined_type("scroll-snap-coordinate",
         use values::CSSFloat;
         use values::computed;
 
+        // https://drafts.csswg.org/css-transforms/#mathematical-description
+        // Note that the matrix elements are addressed in column-major order.
         #[derive(Clone, Copy, Debug, PartialEq)]
         #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
         pub struct ComputedMatrix {
-            pub m11: CSSFloat, pub m12: CSSFloat, pub m13: CSSFloat, pub m14: CSSFloat,
-            pub m21: CSSFloat, pub m22: CSSFloat, pub m23: CSSFloat, pub m24: CSSFloat,
-            pub m31: CSSFloat, pub m32: CSSFloat, pub m33: CSSFloat, pub m34: CSSFloat,
-            pub m41: CSSFloat, pub m42: CSSFloat, pub m43: CSSFloat, pub m44: CSSFloat,
+            pub m11: CSSFloat, pub m21: CSSFloat, pub m31: CSSFloat, pub m41: CSSFloat,
+            pub m12: CSSFloat, pub m22: CSSFloat, pub m32: CSSFloat, pub m42: CSSFloat,
+            pub m13: CSSFloat, pub m23: CSSFloat, pub m33: CSSFloat, pub m43: CSSFloat,
+            pub m14: CSSFloat, pub m24: CSSFloat, pub m34: CSSFloat, pub m44: CSSFloat,
```

Comment:
It's here: https://drafts.csswg.org/css-transforms-2/#three-d-transform-functions

The `matrix3d()` is represented in a column-major way based on the matrix in your link, so it should be `m11, m12, ..., m21, m22, ...`.